chopper/README.md

```diff
@@ -2,21 +2,35 @@
 
 [![pub package](https://img.shields.io/pub/v/chopper.svg)](https://pub.dartlang.org/packages/chopper) [![Build Status](https://travis-ci.com/lejard-h/chopper.svg?branch=master)](https://travis-ci.org/lejard-h/chopper) [![codecov](https://codecov.io/gh/lejard-h/chopper/branch/master/graph/badge.svg)](https://codecov.io/gh/lejard-h/chopper)
 
-[Documentations](https://hadrien-lejard.gitbook.io/chopper)
+[**Documentation**](https://hadrien-lejard.gitbook.io/chopper)
 
-## Installation
+## Adding Chopper to your project
+
+In your project's `pubspec.yaml` file, 
+
+* Add *chopper*'s latest version to your *dependencies*.
+* Add `build_runner: ^1.10.3` to your *dev_dependencies*.
```

Comment:
Should we upgrade the build_runner? :)

Reply:
Could you please test the lib with the latest one?

Also, bumping the `build_runner` version should be a different PR IMHO. 😄 

@JEuler @lejard-h Please note, that I've created a [milestone](https://github.com/lejard-h/chopper/milestone/1) for the next, 3.0.7 release, so please link any further PRs to it on the right until the next release to keep track of the changes easier! 😉 

I'd like to do proper changelogs starting with 3.0.7.

Reply:
Oops, missed that. Of course, this should be a different PR, just asking :) I'm using the fresh build runner in the production app with Chopper - seems okay. :)